### PR TITLE
Tasklist integration tests

### PR DESCRIPTION
**Pull request by wheelsandcogs**
This adds support for running integration tests against workflow.

It uses the `taskflow-test` and `asl-test` (`workflow-asl-test` on `localhost`) databases.

### CODE REVIEW

File: scripts/migrate.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+const migrate = () => {
+  return Promise.resolve()
+    .then(() => Taskflow({ db: config.taskflowDB }).migrate())
+    .catch(err => {
+      console.log(err);
+      process.exit(1);
+    });
+};
+
+migrate();
```
Comment (lennym) [inline scripts/migrate.js]: If I were being pernickety I'd point out that there's no need to wrap code that's executed exactly once in a function. Good job I'm not.

**Comment by lennym** [comment]
It's quite hard to see this and not come up with a lot of ideas of edge cases and situations that we should have test cases for, but as a start this is great.
